### PR TITLE
#158423645 expand ride details to request ride

### PR DESCRIPTION
**Pull request by artorious**
#### What does this PR do?

* Adds passengers' view of an expanded ride offer's details.

* A passenger shuold be able to view all the details associated with that particular ride offer, with an option to confirm/submit the ride requests.

#### Description of Task to be completed?

* Building (without functionality), the UI elements with HTML&CSS. 

#### How the app should be tested
On a webbrowser:

* [Download](https://github.com/artorious/Ride-My-Way/archive/ft-passengers-expanded-ride-158423645.zip) the repo
* Extract and navigate into the `Ride-My-Way` folder
* in the `UI` folder, open `expanded_ride_passenger.html` on a browser

On a  terminal (Linux):

* Install and set up git `sudo apt install git` 

* Clone the repository `git clone https://github.com/artorious/Ride-My-Way.git`

* Run `git checkout ft-passengers-expanded-ride-158423645`

* Run `cd Ride-My-Way/UI/`

* Open `expanded_ride_passenger.html` on a browser

#### Any background context you want to provide?
The UI has no functionality. 
Developed using HTML and CSS

#### What are the relevant pivotal tracker stories?
[#158423645](https://www.pivotaltracker.com/story/show/158423645)

#### Questions:

### CODE REVIEW

**Comment by kwanj-k** [comment]
Great UI

**Comment by xcixor** [comment]
Awesome progress

**Review by LWanjiru — APPROVED** [review]
Consider making your design consistent especially in the `ride details` section. It feels like there is a lot going on in terms of the background colours.
That said good work on the page details.

File: UI/expanded_ride_passenger.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+          <input type="submit" value="Request ride">
+        </div>
+      </fieldset>
+    </form>
+  </div>
+  <hr>
+  <footer>&copy;RideMyWay 2018.<br><a href="#">Contact Us.</a></footer>
+</div>
+</body>
+</html>
```
Comment (LWanjiru) [inline UI/expanded_ride_passenger.html]: add new line here

**Comment by artorious** [comment]
@kwanj-k @xcixor @LWanjiru Thanks for the input. I've made adjustments to make the page easier on the eyes

exprdidepass

**Pull request merged**